README.md

```diff
@@ -1041,6 +1041,42 @@ endpoints:
 ```
 
 
+#### Configuring Ses alerts
+| Parameter                      | Description                                                                                | Default       |
+|:-------------------------------|:-------------------------------------------------------------------------------------------|:--------------|
+| `alerting.ses`                 | Settings for alerts of type `ses`                                                          | `{}`          |
```

Comment:
Could you modify the alerting provider key to `aws-ses`? (e.g. alerting.aws-ses)